### PR TITLE
TokenRequest and TokenRequestProjection are GA now

### PR DESCRIPTION
**Pull request by zshihang**
TokenRequest and TokenRequestProjection are GA now

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -286,15 +286,16 @@ TODO: Test and explain how to use additional non-K8s secrets with an existing se
...
 {{< note >}}
-This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+This TokenRequest and TokenRequestProjection are __GA__ in 1.20 and
 enabled by passing all of the following flags to the API server:
 
 * `--service-account-issuer`
+* `--service-account-key-file`
 * `--service-account-signing-key-file`
-* `--service-account-api-audiences`
+* `--api-audiences`
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:298]: GA features are enabled by default. We can omit the note.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -153,8 +153,10 @@ different Kubernetes components.
 | `Sysctls` | `true` | Beta | 1.11 | |
 | `TokenRequest` | `false` | Alpha | 1.10 | 1.11 |
 | `TokenRequest` | `true` | Beta | 1.12 | |
+| `TokenRequest` | `true` | GA | 1.20 | |
 | `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
 | `TokenRequestProjection` | `true` | Beta | 1.12 | |
+| `TokenRequestProjection` | `true` | GA | 1.20 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: Please move these feature gates into the next section; they no longer belong under the heading _Feature gates for Alpha or Beta features_.

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks @zshihang 

I've added some advice about making these changes match how SIG Docs normally does them.

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 3]
```diff
@@ -286,15 +286,16 @@ TODO: Test and explain how to use additional non-K8s secrets with an existing se
 
 ## Service Account Token Volume Projection
 
-{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
 
 {{< note >}}
-This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+This TokenRequest and TokenRequestProjection are __GA__ in 1.20 and
 enabled by passing all of the following flags to the API server:
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:294]:
```suggestion
To enable and use token request projection, you must specify each of the following
command line arguments to `kube-apiserver`:

```

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:298]: [Suggested text](https://github.com/kubernetes/website/pull/24823/files#r515684800) added.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 2 above)
Comment (zshihang) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:159]: done

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md (hunk 3 above)
Comment (zshihang) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:294]: done

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md (hunk 1 above)
Comment (zshihang) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md]: thanks

**Comment by kikisdeliveryservice** [comment]
As noted on this PR (https://github.com/kubernetes/kubernetes/pull/93258#issuecomment-720622085) the release team has no Enhancement 1205 on record as being GA for this release. Please clarify/file and exception request as necessary.

**Comment by zshihang** [comment]
exception issue is opened https://github.com/kubernetes/enhancements/issues/2126

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -303,6 +303,8 @@ different Kubernetes components.
 | `TaintNodesByCondition` | `false` | Alpha | 1.8 | 1.11 |
 | `TaintNodesByCondition` | `true` | Beta | 1.12 | 1.16 |
 | `TaintNodesByCondition` | `true` | GA | 1.17 | - |
+| `TokenRequest` | `true` | GA | 1.20 | - |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:306]: Do NOT remove the history.
Line 155-158 should be moved here.
The goal is to let users see the evolution history of the feature, not just its current state.

**Comment by annajung** [comment]
/hold cancel
exception approved, k/k pr merged

**Comment by liggitt** [comment]
/hold

would like to keep this on hold until KEP updates are made

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 4 above)
Comment (zshihang) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:305]: done. the history was not removed but kept in the last section.

**Comment by zshihang** [comment]
KEP updates merged. https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md

/hold cancel

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**